src/intrinsics/prepack/global.js

```diff
@@ -129,15 +129,49 @@ export default function(realm: Realm): void {
   //     that is not subsequently applied, the function will not be registered
   //     (because prepack won't have a correct value for the FunctionValue itself)
   global.$DefineOwnProperty("__optimize", {
-    value: new NativeFunctionValue(realm, "global.__optimize", "__optimize", 1, (context, [value, config]) => {
+    value: new NativeFunctionValue(realm, "global.__optimize", "__optimize", 1, (context, [value, argModelString]) => {
       // only optimize functions for now
+      if (argModelString !== undefined) {
+        let argModelError;
+        if (argModelString instanceof StringValue) {
+          try {
+            JSON.parse(argModelString.value);
```

Comment:
Maybe add a comment here in the code that we intentionally ignore the result as we are just trying to produce the error message here.